### ISSUE TITLE
Unable to run Docker image on M1 Mac

### ISSUE THREAD

**Issue by davorb** [oblador/loki#344]
I'm trying to run Loki with

```
    "configurations": {
      "chrome.laptop": {
        "target": "chrome.docker",
      }
    }
```

It was previously mentioned [in this thread](https://github.com/oblador/loki/issues/309). Which made me try to run

```
docker run --rm -P --security-opt=seccomp=$CWD/node_modules/@loki/target-chrome-docker/src/docker-seccomp.json --shm-size=1g yukinying/chrome-headless-browser:90.0.4408.0 --disable-datasaver-prompt --no-first-run --disable-extensions --remote-debugging-address=0.0.0.0 --remote-debugging-port=61867 --headless --disable-gpu --hide-scrollbar
```

This fails to run on an M1 Mac with the error message:
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
clone: Invalid argument
qemu: uncaught target signal 4 (Illegal instruction) - core dumped
```

Any ideas of what I can do to make it run on my machine?

**Comment by glebez** [comment]
Here's a link to the discussion in the docker image repo, related to this issue: https://github.com/yukinying/chrome-headless-browser-docker/issues/46

**Comment by rwieruch** [comment]
@cesar-carrera-nutmeg maybe a stupid question, but if I am using Docker Desktop these days, is Colima a substitute or do I need to marry both tools somehow?

**Comment by oblador** [comment]
Fixed in [0.30.0](https://github.com/oblador/loki/releases/tag/v0.30.0)
